### PR TITLE
[FrameworkBundle] Add phpstorm ide

### PR DESCRIPTION
**Pull request by hason**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

Works on Windows, Mac (https://github.com/aik099/PhpStormProtocol) and Linux (https://github.com/sanduhrs/phpstorm-url-handler/commit/8f7a8dff04c0fb68d81c52fdef4c410548a3324b).

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -100,6 +100,7 @@ public function load(array $configs, ContainerBuilder $container)
                     'macvim' => 'mvim://open?url=file://%%f&line=%%l',
                     'emacs' => 'emacs://open?url=file://%%f&line=%%l',
                     'sublime' => 'subl://open?url=file://%%f&line=%%l',
+                    'phpstorm' => 'pstorm://open/?url=file://%%f&line=%%l',
```
Comment (enumag) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: Is the slash before "?" necessary?

Comment (hason) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: I created PR https://github.com/aik099/PhpStormProtocol/pull/19

Comment (hason) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: Fixed in https://github.com/aik099/PhpStormProtocol/pull/20

**Comment by fabpot** [comment]
Thank you @hason.

**Pull request merged**